cert/selfsigned.go

```diff
@@ -72,10 +75,11 @@ func ipAddresses(tlsExtraIPs []string) ([]net.IP, error) {
 
 // dnsNames returns the host and DNS names to use when creating the TLS
 // ceftificate.
-func dnsNames(tlsExtraDomains []string) (string, []string) {
+func dnsNames(tlsExtraDomains []string, TLSDisableAutofill bool) (string, []string) {
 	// Collect the host's names into a slice.
 	host, err := os.Hostname()
-	if err != nil {
+	// If TLSDisableAutofill is true, don't use the hostname
```

Comment:
nit: add an empty line before any comment and always write full sentences (capitalized and with full stop). Also, the _why_ is more interesting than explaining the code.